client/packages/common/src/intl/locales/fr/common.json

```diff
@@ -583,5 +583,7 @@
   "label.dose-number": "Dose #",
   "label.label": "Etiquette",
   "error.field-must-be-specified": "{{field}} doit être spécifié",
-  "button.add-new-vaccine-course": "Ajouter un calendrier de vaccination"
-}
+  "button.add-new-vaccine-course": "Ajouter un calendrier de vaccination",
+  "message.arguments": "Vous pouvez personnaliser l'affichage du rapport en spécifiant certains des détails facultatifs ci-dessous. Cliquez sur OK pour afficher le rapport.",
```

Comment:
With it in common.json?

![Screenshot 2024-10-02 at 3 53 55 PM](https://github.com/user-attachments/assets/7f8e28f9-94d8-4b4e-ad08-8ccf7ff1d26d)

Works if I move to reports.json: https://github.com/msupply-foundation/open-msupply/compare/105-report-translations...105-report-translations(move-arguments-message-translation)

Not the same for you?

Reply:
Committed this change to try get build underway ASAP, sorry 🙏